master/reference/node/configuration.md

```diff
@@ -40,7 +40,13 @@ In addition to the above, `calico/node` also supports [the standard Felix config
 
 ### IP Autodetection methods
 
-The following describe the available IP autodetection methods.
+When Calico starts up it must select an interface which will be used for all
```

Comment:
I wouldn't say when it starts it must select an interface, since a perfectly valid approach is to explicitly configure the IP address and then there is no selection required.

I'd say something more like:

When Calico is used for routing, each node must be configured with the IPv4 address (and IPv6 address if using IPv6) that would be used to route between nodes.  To reduce the amount of configuration that needs to be explicitly provided, the calico/node container can be configured to autodetect the IP addresses to use for routing.  In many systems, there might be multiple physical interfaces on a host, or possibly multiple IP addresses configured on a physical interface.  In these cases, there are multiple addresses to choose from and so autodetection of the correct address can be tricky.

The IP autodetection methods are provided to improve the selection of the correct address, by limiting the selection based on suitable criteria for your deployment."